utils/build-toolchain

```diff
@@ -98,15 +98,15 @@ set -x
 YEAR=$(date +"%Y")
 MONTH=$(date +"%m")
 DAY=$(date +"%d")
-TOOLCHAIN_VERSION="swift-LOCAL-${YEAR}-${MONTH}-${DAY}-a"
+TOOLCHAIN_VERSION="5.0.${YEAR}${MONTH}${DAY}"
```

Comment:
Note: the toolchain version format was copied from `Info.plist` from macOS toolchains from https://swift.org/download.

The leading `5.0` component appears to refer to the current Swift version (which incidentally is no longer 5.0 for `master` `HEAD`).

I'm not sure how to make it programmatically refer to the current Swift version. Hardcoding it to `5.0` seems fine for now.

Reply:
We could generate this file from CMake.  There is no way to do this currently.  I think that this is reasonable.